with-inbox-sdk/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "with-inbox-sdk",
+  "displayName": "With inbox sdk",
+  "version": "0.0.0",
+  "description": "Plasmo extension which uses inbox sdk for building browser extensions for Gmail",
+  "author": "yashwadhia",
+  "scripts": {
+    "dev": "plasmo dev",
+    "build": "plasmo build"
+  },
+  "dependencies": {
+    "@inboxsdk/core": "0.2.7",
+    "@types/inboxsdk": "2.0.8",
+    "plasmo": "latest",
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
+  },
+  "devDependencies": {
+    "@trivago/prettier-plugin-sort-imports": "3.2.0",
+    "@types/chrome": "0.0.191",
+    "@types/node": "18.0.0",
+    "@types/react": "18.0.14",
+    "@types/react-dom": "18.0.5",
+    "prettier": "2.7.1",
+    "typescript": "4.7.4"
+  },
+  "manifest": {
+    "host_permissions": [
+      "https://*/*"
+    ],
+    "permissions": [
+      "scripting"
+    ],
+    "content_scripts": [
```

Comment:
We don't need this, use host_permissions instead for the matches.

In term of content-script, uses: 

https://docs.plasmo.com/workflows/content-scripts#injecting-ui-elements

Reply:
Okay let me take care of this part